common/lz_compress.py

```diff
@@ -131,10 +131,16 @@ def _compress(input):
 
     window = SlidingWindow(input)
 
+    percentage = 0
+
     i = 0
     while True:
         if len(input) <= i:
             break
+        new_percentage = int(i / len(input) * 100)
+        if percentage != new_percentage:
+            print("{}%".format(new_percentage))
```

Comment:
just ultimately nitpicking here: `print("{}%\r".format(new_percentage), end="")` ? ;-)

Reply:
Couldn't get this working with Travis